src/platform/qt/ConfigController.cpp

```diff
@@ -153,7 +156,12 @@ ConfigController::ConfigController(QObject* parent)
 	m_subparsers[1].usage = "Frontend options:\n"
 	    "  --ecard FILE  Scan an e-Reader card in the first loaded game\n"
 	    "                Can be passed multiple times for multiple cards\n"
-	    "  --mb FILE     Boot a multiboot image with FILE inserted into the ROM slot";
+	    "  --mb FILE     Boot a multiboot image with FILE inserted into the ROM slot"
+#ifdef ENABLE_SCRIPTING
+	    "  --script FILE Lua script to load on start"
```

Comment:
```suggestion
	    "\n  --script FILE Script file to load on start"
```

This needs a newline at the beginning so it doesn't get concatenated to the previous line. Additionally, scripting languages beyond Lua are planned, so let's not be specific here.